SU-SWS/stanford_capx
# DEVOPS-256: CAPx fatal error with cardinald7 1.0.41 release

# READY FOR REVIEW

# Summary
- Update `guzzle` and other dependencies

# Needed By (Date)
- Will be required to push a new version of `cardinald7` and `leland` to get the latest version of `acquia_purge_d8cache` deployed (because the last point release had to be rolled back)

# Criticality
- How critical is this PR on a 1-10 scale? 7/10
- CAPx is broken without it

# Steps to Test

1. Run through the [CAPx Behat tests](https://github.com/SU-SWS/linky_clicky/tree/5.x/includes/features/SU-SWS/stanford_capx)
    1. Difficulty: I'm having trouble getting the "[And I am authenticated with CAPx](https://github.com/SU-SWS/linky_clicky/blob/5.x/includes/features/SU-SWS/stanford_capx/1st_stanford_capx.feature#L9)" step to work.
2. Check out the `DEVOPS-256` branches on ACSF `cardinald7` and `leland` stacks and test as above

# Affected Projects or Products
- CAPx D7
- ACSF cardinald7 and leland

# Associated Issues and/or People
- JIRA ticket: DEVOPS-256

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
